dashboard/src/reducers/charts.ts

```diff
@@ -75,6 +78,8 @@ const chartsReducer = (state: IChartState = initialState, action: ChartsAction):
       return { ...state, selected: chartsSelectedReducer(state.selected, action) };
     case getType(actions.charts.selectValues):
       return { ...state, selected: chartsSelectedReducer(state.selected, action) };
+    case getType(actions.charts.selectSchema):
+      return { ...state, selected: chartsSelectedReducer(state.selected, action) };
```

Comment:
I don't know the context here, but the above 10 lines look like they could avoid the redundancy with:
```
case getType(actions.charts.resetChartVersion):
case getType(actions.charts.selectReadme):
case getType(actions.charts.errorReadme):
case getType(actions.charts.errorChart):
case getType(actions.charts.selectValues):
case getType(actions.charts.selectSchema):
  return { ...state, selected: chartsSelectedReducer(state.selected, action) };
```